.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = tab
+indent_size = 4
+
+[*.md]
+indent_style = space
+trim_trailing_whitespace = false
+
+[*.yml]
+indent_style = space
+indent_size = 2
```

Comment:
we don't really have files like that anywhere else, so I am not particularly happy with adding this with this PR specifically

Reply:
I'm fine with the dropping :smile: